src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/BasicFieldsTest.java

```diff
@@ -25,9 +25,14 @@ public BasicFieldsTest() {
         super("/org/jboss/xavier/analytics/rules/workload/inventory/BasicFields.drl", ResourceType.DRL);
     }
 
-    @Test
-    public void test() throws ParseException {
+    private void checkLoadedRulesNumber()
```

Comment:
What do you think about annotating with @Before as you are running it before each test.

Reply:
Thanks for the idea: it made me think about this "check" and i decided to "promoted" it to become a test